### PR TITLE
Feature: Allow controlling the html-preview frame by rendering with pure fusion

### PR DESCRIPTION
**Pull request by mficzel**
The setting `Sitegeist.Monocle.preview.fusionRootPath` defines a fusion path
that is rendered as HTML-Frame around a component.

The default value is `/<Sitegeist.Monocle:Preview.Page>`
The context present during rendering contains only `html`, `sitePackageKey` 
and `prototypeName` 

The fusion-prototype `Sitegeist.Monocle:Preview.Page` uses an api similar to
`Neos.Fusion:Page` but is based on `Neos.Fusion:Array` and thus avoids the
magic of `Neos.Neos:Page` or `Neos.Fusion:Http.Message`.

The Prototypes `Sitegeist.Monocle:Preview.Header.MetaViewport`,
`Sitegeist.Monocle:Preview.Header.Styles` and
`Sitegeist.Monocle:Preview.Header.Scripts` are added to render header scripts
based on the preview configuration. They are mainly there to be used inside the 
preview but can be used in the main-page aswell.

### CODE REVIEW

File: Resources/Private/Fusion/Preview/Header/Viewport.fusion [hunk 1]
```diff
@@ -0,0 +1,12 @@
+##
+# Prototype that renders a meta-vireport tag based on the setting Sitegeist.Monocle.preview.metaViewport
```
Comment (grebaldi) [inline Resources/Private/Fusion/Preview/Header/Viewport.fusion]: "meta-vireport" -> "meta-viewport"

Comment (mficzel) [inline Resources/Private/Fusion/Preview/Header/Viewport.fusion]: Das ist angepasst

File: Resources/Private/Fusion/Preview/Header/Scripts.fusion [hunk 2]
```diff
@@ -0,0 +1,16 @@
+##
+# Prototype that renders a list of script-tags based on the setting Sitegeist.Monocle.preview.metaViewport
```
Comment (hlubek) [inline Resources/Private/Fusion/Preview/Header/Scripts.fusion]: s/metaViewPort/additionalResources.javaScripts

File: README.md [hunk 3]
```diff
@@ -121,6 +121,9 @@ Sitegeist:
     preview:
       # the path the styleguide will show at start    
       defaultPath: 'atoms'
+      # fusion root path that is used to render the frame around the component
+      # the fusion gets the context-variables `html`, `sitePackageKey` and `prototypeName
```
Comment (hlubek) [inline README.md]: `prototypeName` (missing closing backtick)

File: Resources/Private/Fusion/Preview/Header/Styles.fusion [hunk 4]
```diff
@@ -0,0 +1,16 @@
+##
+# Prototype that renders a list of style-tags based on the setting Sitegeist.Monocle.preview.metaViewport
```
Comment (hlubek) [inline Resources/Private/Fusion/Preview/Header/Styles.fusion]: s/metaViewPort/additionalResources.styleSheets

**Comment by jonnitto** [comment]
Really, really great. This allows me to include some inline svg for icons…
It would be awesome if you could merge and release this feature as soon as possible…

**Comment by mficzel** [comment]
I close this in favor of https://github.com/sitegeist/Sitegeist.Monocle/pull/37 that has the fusion-controlled preview aswell but with a much deeper integration.

**Pull request closed without merging**